docker-compose.yml

```diff
@@ -34,6 +34,8 @@ services:
   #   image: minio/minio
   #   ports:
   #     - "9000:9000"
-  #   env_file:
-  #     - build/docker.env
-  #   command: server /data
+  #     - "9001:9001"
```

Comment:
A useful "trick", if the exposed port are not frequently used, you can expose them to random by 
```
":9000"
":9001"
```

This way port collision can be avoided (saying that because 9000 is a popular port)